pyigm/cgm/cgm.py

```diff
@@ -323,10 +330,26 @@ def stack_plot(self,to_plot,pvlim=None,maxtrans=3,return_fig=True,**kwargs):
                 else: # Pick components in system closest to z_cgm
                     thesecomps = pu.get_components(self,tp)
                     if len(thesecomps)==0:
-                        continue
-                    # Find component with redshift closest to systemic
-                    compvels = np.array([np.median(tc.vlim.value) for tc in thesecomps])
-                    comp = thesecomps[np.argmin(np.abs(compvels))]
+                        if add_missing_lines is True:
+                            if isinstance(tp,str):
+                                tup = ltai.name_to_ion(tp)
+                            else:
+                                tup = tp
+                            comp = AbsComponent(self.coord,tup,zcomp=self.z,
+                                                vlim=[-100.,100.]*u.km/u.s)
+                            comp.add_abslines_from_linelist()
+                            if spec is not None:
```

Comment:
Maybe add a comment here.  I can't remember why the
first element is enough for setting ```spec```

Reply:
Good catch!  It isn't enough.